src/functionalTest/resources/scenarios/RIA-3378-expected-400-response-invalid-format.json

```diff
@@ -0,0 +1,28 @@
+{
+  "description": "RIA-3378 expected 400 response for appeal reference invalid format",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "LegalRepresentative",
+    "input": {
+      "eventId": "submitAppeal",
+      "state": "appealSubmitted",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "homeOfficeReferenceNumber": "1212/009900362015"
```

Comment:
Are these the home office provided id's?

Reply:
That's correct @raghera . All of these IDs are based on the different scenarios to test against PRP1